src/modules/neo4j/neo4j-container.ts

```diff
@@ -30,6 +30,10 @@ export class Neo4jContainer extends GenericContainer {
     );
   }
 
+  public withTtl(schedule = 5) {
```

Comment:
was this not already merged into master?

Reply:
Yes, this is weird. I can't see any difference here. Do you know how to remove this from the pull? When I checkout the file from the base branch it doesn't show any modificationn

Reply:
seems there's a conflict here now

Reply:
I don't get a conflict here